corehq/ex-submodules/dimagi/utils/couch/sync_docs.py

```diff
@@ -35,7 +35,7 @@ def index_design_docs(db, docid, design_name):
             try:
                 list(db.view(view, limit=0))
             except RequestFailed as e:
-                if 'timeout' not in e.message:
```

Comment:
is it possible to check if the status code is 504?

Reply:
You could try checking `e.status_int` ... but I'm not sure whether it'll be set when the exception is raised. 

(I also noticed that a RequestTimeout exception exists. Interesting that it doesn't get raised. It extends `Exception` and doesn't have a `status_int` property.)